lib/pages/new_private_chat/create_room_controller.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/model/room/new_room_request.dart';
+import 'package:fluffychat/domain/usecase/create_room_interactor.dart';
+import 'package:matrix/matrix.dart';
+
+class CreateRoomController {
+  final _createRoomInteractor = getIt.get<CreateRoomInteractor>();
+  final streamController = StreamController<Either<Failure, Success>>();

```

Comment:
I dont know why we need another `stream` to only clone the event of other stream. Really hard to understand @imGok @sherlockvn

Reply:
will have a huge number of duplicated stream like that in the future.

Reply:
IMO, we need to wrap in streamController because we need to manage the lifecycle ourselves

Reply:
no need, we only listen the `interactor`, then all will be disposed when the controller disposes.